pkg/core/blockchain.go

```diff
@@ -617,6 +621,20 @@ func (bc *Blockchain) storeBlock(block *block.Block, txpool *mempool.Pool) error
 			return fmt.Errorf("failed to store tx exec result: %w", err)
 		}
 		writeBuf.Reset()
+
+		if bc.config.P2PSigExtensions {
+			for _, attr := range tx.GetAttributes(transaction.ConflictsT) {
+				hash := attr.Value.(*transaction.Conflicts).Hash
+				dummyTx := transaction.NewTrimmedTX(hash)
+				dummyTx.Version = transaction.DummyVersion
+				dummyTx.Script = []byte{0x0} // avoid decoding errors
```

Comment:
We can make `DummyVersion` transaction encode and decode with just `version` field to simplify this and make DB value more compact.